taskomatic/jobs/usage_metrics.py

```diff
@@ -0,0 +1,291 @@
+"""
+Feature and product usage metrics.
+"""
+import io
+import datetime
+import time
+from functools import wraps
+
+from prometheus_client import Gauge
+from psycopg2.extras import execute_values
+
+from common.config import Config
+from common.database_handler import DatabasePool, DatabasePoolConnection, NamedCursor
+from common.logging import init_logging, get_logger
+
+LOGGER = get_logger(__name__)
+
+CFG = Config()
+
+MAX_RESULTS_COUNT = 20
+SYSTEM_NUMS = [50, 100, 500, 1000]
+
+CVE_STATUS_USAGE_T = Gauge("ve_usage_cve_status_t", "Total number of cve status usage", ["account"])
+CVE_STATUS_USAGE_TF = Gauge("ve_usage_cve_status_tf", "Total number of cve status usage without RH accounts", ["account"])
+SYSTEM_CVE_STATUS_USAGE_T = Gauge("ve_usage_system_cve_status_t", "Total number of system cve status usage", ["account"])
+SYSTEM_CVE_STATUS_USAGE_TF = Gauge("ve_usage_system_cve_status_tf", "Total number of system cve status usage without RH accounts", ["account"])
+CVE_BRISK_USAGE_T = Gauge("ve_usage_cve_business_risk_t", "Total number of business risk usage", ["account"])
+CVE_BRISK_USAGE_TF = Gauge("ve_usage_cve_business_risk_tf", "Total number of business risk usage without RH accounts", ["account"])
+BIG_ACCOUNTS_COUNT_F = Gauge("ve_usage_big_accounts_count_f", "Count of accounts with systems without RH accounts", ["systems"])
+
+
+def timer(report_name):
+    """Counts the time spent on the request"""
+
+    def _decorator_timer(func):
+        @wraps(func)
+        def _wrapper_timer(*args, **kwargs):
+            start = time.time()
+
+            LOGGER.debug("Generating %s report...", report_name)
+            try:
+                return func(*args, **kwargs)
+            finally:
+                end = (time.time() - start) * 1000
+
+                LOGGER.debug("-----------------------------------")
+                LOGGER.info("%s query took %s milliseconds", report_name, str(end))
+
+        return _wrapper_timer
+
+    return _decorator_timer
+
+
+class StatsManager:
```

Comment:
I think whole stats manager class can be removed as we no longer are storing data in DB and this is class connecting DB and code.